### ISSUE TITLE
[bug] Error while installing webp

### ISSUE THREAD

**Issue by karambaq** [conan-io/conan#14125]
### Environment details

* Operating System+version: MacOS 13.2.1 
* Python version: 3.9

```
> pip3 install webp==0.1.3
Defaulting to user installation because normal site-packages is not writeable
Collecting webp==0.1.3
  Using cached webp-0.1.3.tar.gz (10 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [33 lines of output]
      Traceback (most recent call last):
        File "/Users/malter/Library/Python/3.9/lib/python/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/Users/malter/Library/Python/3.9/lib/python/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/Users/malter/Library/Python/3.9/lib/python/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 149, in prepare_metadata_for_build_wheel
          return hook(metadata_directory, config_settings)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 380, in prepare_metadata_for_build_wheel
          self.run_setup()
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 10, in <module>
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 107, in setup
          return distutils.core.setup(**attrs)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 147, in setup
          _setup_distribution = dist = klass(attrs)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 496, in __init__
          _Distribution.__init__(
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 283, in __init__
          self.finalize_options()
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 935, in finalize_options
          ep(self)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 955, in _finalize_setup_keywords
          ep.load()(self, ep.name, value)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 219, in cffi_modules
          add_cffi_module(dist, cffi_module)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 49, in add_cffi_module
          execfile(build_file_name, mod_vars)
        File "/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/cffi/setuptools_ext.py", line 25, in execfile
          exec(code, glob, glob)
        File "webp_build/builder.py", line 7, in <module>
          from conans.client import conan_api
      ImportError: cannot import name 'conan_api' from 'conans.client' (/private/var/folders/lr/yvn7d23x3cq5kkk1180tf1cc0000gn/T/pip-build-env-0h3fl_1z/overlay/lib/python3.9/site-packages/conans/client/__init__.py)
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

### Steps to reproduce

pip3 install webp==0.1.3

**Comment by memsharded** [comment]
Hi @karambaq 

I think this should be reported to https://github.com/anibali/pywebp. It doesn't seem a Conan error, but an issue with that project, maybe it needs pinning a dependency

**Comment by memsharded** [comment]
pywebp limits Conan version <2.0 https://github.com/anibali/pywebp/blob/master/pyproject.toml, is it possible that you already have Conan 2.0 in the environment and it is colliding?

**Comment by karambaq** [comment]
@memsharded 

I don't have conan installed in environment:

`pip3 list | grep conan` is empty

**Comment by memsharded** [comment]
The maintainer of webp is discouraging the usage of ``0.1.3`` and recommends updating to latest ``0.1.5`` to solve this issue in https://github.com/anibali/pywebp/issues/39

**Comment by memsharded** [comment]
https://github.com/anibali/pywebp/issues/39 closed, the latest ``0.1.6`` seems to be working, this is not a Conan issue per-se, thanks for the feedback!
